src/repr/src/lib.rs

```diff
@@ -127,3 +127,57 @@ pub use crate::scalar::{
     PropDict, PropList, ProtoScalarType, ScalarBaseType, ScalarType,
 };
 pub use crate::timestamp::{Timestamp, TimestampManipulation};
+
+pub mod statement_logging {
```

Comment:
This is subjective, but why not put this in a `statement_logging.rs` file?

Reply:
I was planning on doing that and forgot to do it before putting this PR up.